interface/repository/centre_repository.go

```diff
@@ -0,0 +1,18 @@
+package repository
+
+import (
+	"api-booking-time/domain/model"
+	"api-booking-time/usecase/repository"
+)
+
+type centreRepository struct {
+	centres *[]*model.Centre
```

Comment:
It's the first time I see a pointer to an array of pointers.

Not sure if this is the best approach. 

Generally speaking, a pointer to an array or an array of pointers would be enough for most cases, but don't come to mind why you would use this.

Would you mind elaborating on this?